helm/efk-stack-app/charts/opendistro-es/templates/elasticsearch/es-client-ingress.yaml

```diff
@@ -42,8 +46,11 @@ spec:
       http:
         paths:
           - path: {{ $ingressPath }}
+            pathType: ImplementationSpecific
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: http
+              service: 
```

Comment:
Looks like you need to do the if/else for the ingress apiVersion also here. `networking.k8s.io/v1beta1`'s backend definition is the same as the one in extensions. See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#ingressbackend-v1beta1-networking-k8s-io

Reply:
Please review